telegram_bot/utils/api.py

```diff
@@ -0,0 +1,57 @@
+import os
+import uuid
+
+import requests
+from dotenv import load_dotenv
+
+from utils.format import format_chat_text
+
+
+load_dotenv()
+backend_url = os.getenv("BACKEND_URL")
+
+
+def search_schemes(text: str, similarity_threshold: int) -> tuple[str, list | None, str | None]:
+    """
+    Handles API call to backend (searching for schemes)
+    """
+
+    query_id = str(uuid.uuid4())
+    body = {"sessionID": query_id, "query": text, "similarity_threshold": similarity_threshold}
+
+    endpoint = backend_url + "/schemespredict"
+
+    res = requests.post(endpoint, json=body)
+
+    if res.status_code != 200:  # Error
+        err_message = "I am unable to search for suitable assistance schemes. Please try again!"
+        return (query_id, None, err_message)
+
+    schemes = res.json()["data"]
+
+    if len(schemes) == 0:  # No suitable schemes found
```

Comment:
```suggestion
    if not schemes:  # No suitable schemes found
```
follow [pep8](https://peps.python.org/pep-0008/)
> Don’t check for empty containers or sequences (like [] or '') by comparing the length to zero (if len(somelist) == 0). Use if not somelist and assume that empty values will implicitly evaluate to False.